Valet.xcodeproj/xcshareddata/xcschemes/Valet watchOS Test Host App Extension.xcscheme

```diff
@@ -91,17 +89,6 @@
             ReferencedContainer = "container:Valet.xcodeproj">
          </BuildableReference>
       </RemoteRunnable>
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "16DF6AFE204B496800F8E0A4"
-            BuildableName = "Valet watchOS Test Host App Extension.appex"
-            BlueprintName = "Valet watchOS Test Host App Extension"
-            ReferencedContainer = "container:Valet.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
```

Comment:
Why did this one and `Valet watchOS Test Host App` disappear?

Reply:
The `MacroExpansion` disappeared but the `RemoteRunnable` is still there. I don't know what the former is.